README.md

```diff
@@ -138,12 +145,109 @@ make test-ts-binary-rococo
 
 Remember to run the clean-up afterwards.
 
-## extend the leasing period
+## License
 
-The default leasing duration for parachain is 1 day, in case you want to extend it (even after it's downgraded to parathread), simply do a `forceLease` via sudo, it should be upgraded to parachain soon again and start to produce blocks.
+GPLv3
 
-![image](https://user-images.githubusercontent.com/7630809/135689832-1f57cd5c-7f83-4fce-9bb0-832b77a38dcc.png)
 
-## License
+## How to Build and Run Parachain and Tee-worker
```

Comment:
I believe we have "launch a parachain network" in the previous section already

Reply:
It's been updated since but I didn't merge as I was waiting for the feedback from Faisal.